requirements-m1.txt

```diff
@@ -0,0 +1,6 @@
+tokenizers
+transformers
+fastapi==0.63.0
```

Comment:
Seems like `sentencepiece` goes unused so I removed it. I only noticed because it was a serious PITA to install it on mac. 

Suggest removing it from the main requirements file as well